src/draive/evaluators/text_keywords.py

```diff
@@ -0,0 +1,9 @@
+from draive.evaluation import evaluator
+
+__all__ = [
+    "text_keywords_evaluator",
+]
+
+@evaluator(name="keywords", threshold=0.95)
+async def text_keywords_evaluator(text: str, keywords: list[str]) -> float:
```

Comment:
please add comma after the last argument

Reply:
Such behavior should be enforced by the formatter. Usually it's only enabled for multi-line.